.github/workflows/firefox-ios-autofill-playwrite-tests.yml

```diff
@@ -39,3 +49,23 @@ jobs:
             
             echo "Run tests"
             npm test
+        - name: Send custom JSON data to Slack workflow
+          if: always()
+          id: slack
+          uses: slackapi/slack-github-action@v1.26.0
+          with:
+            payload: |
+              {
+                "text": "GitHub Action Running Tests For ${{ env.github_action_name }} result: ${{ job.status }}",
+                "blocks": [
+                  {
+                    "type": "section",
+                    "text": {
+                      "type": "mrkdwn",
+                      "text": "GitHub Action Running Tests For result: ${{ job.status }}"
+                    }
+                  }
+                ]
+              }
+          env:
+            SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
I created this secret from my fork, I guess I would need you @nbhasin2 to create the same in firefox repo?. I can share the value via 1Password link